.gitignore

```diff
@@ -127,4 +127,7 @@ dist
 **/*/iife.json
 **/*/module.json
 
+# ignore polyfills copy
+**/*/polyfills
```

Comment:
Since this is copied to a predictable location per package, recommend:

```suggestion
packages/*/src/polyfills
```

Reply:
ah, yup, that makes sense. Otherwise, potential new files in shared/polyfills would also be ignored.